exploration/0002-expression-attributes.md

```diff
@@ -38,18 +38,70 @@ At least the following expression attributes should be considered:
 
   - `fallback` — A value to use instead of the default fallback,
     should the expression's primary formatting fail in some way.
+
+    > Example that would format as "Hello world" if `$place` is not defined:
+    >
+    > ```
+    > {Hello {$place @fallback=world}}
+    > ```
+
+  - `id` — An identifier for the expression.
+    This is included in the formatted part,
+    and allows the parts of an expression to be explicitly addressed.
+
+    > Example identifying two formatted numbers
+    > in a message with badly named input variables:
```

Comment:
Perhaps use better names for the input variables? Or are you thinking of MF1 ported values with auto-numbered replacements?

I think this use case is more compelling when the variable names and ID names align differently, such as when the formatted value needs to be placed withing a DOM element with a given ID but the input variable names cannot be changed.

Reply:
Replaced the example to show how `@id` can make placeholders formatting literal values identifiable.